SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -27,6 +27,11 @@ const FString FSpatialTestSettings::GeneratedOverrideSettingsDirectory =
 	FPaths::ConvertRelativePathToFull(FPaths::ProjectIntermediateDir() / TEXT("Config/") / OverrideSettingsFileDirectoryName);
 const FString FSpatialTestSettings::GeneratedOverrideSettingsBaseFilename = GeneratedOverrideSettingsDirectory / OverrideSettingsFilePrefix;
 
+FString FSpatialTestSettings::GenerateMapConfigurationFilename(const FString& MapName)
```

Comment:
```suggestion
/*static*/ FString FSpatialTestSettings::GenerateMapConfigurationFilename(const FString& MapName)
```
Minor: this is a convention that is used in the gdk.